src/Package/OperationAnalyser.php

```diff
@@ -6,25 +6,20 @@
 namespace Vaimo\ComposerPatches\Package;
 
 use Composer\DependencyResolver\Operation\OperationInterface;
+use Vaimo\ComposerPatches\Plugin;
 
 class OperationAnalyser
 {
-    /**
-     * @var \Vaimo\ComposerPatches\Package\ConfigAnalyser
-     */
-    private $configAnalyser;
-
-    public function __construct()
-    {
-        $this->configAnalyser = new \Vaimo\ComposerPatches\Package\ConfigAnalyser();
-    }
-
     public function isPatcherUninstallOperation(OperationInterface $operation)
     {
         if (!$operation instanceof \Composer\DependencyResolver\Operation\UninstallOperation) {
             return false;
         }
 
-        return $this->configAnalyser->ownsNamespace($operation->getPackage(), __NAMESPACE__);
+        return \in_array(
```

Comment:
Have you looked at adding a test for this change?

Perhaps there is some existing test, which exercises this code, that could be the basis for an additional test to show that the "Vaimo namespace case" works as expected?

Reply:
I gave up on figuring out how it is tested. It has too smart testing framework and close to no documentation on how to actually use it :(